README.md

```diff
@@ -1956,6 +1956,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
   - [Tabassum Fatima](https://github.com/tab21)
   - [Tadi](https://github.com/mtrill47)
+  - [Taimoor](https://github.com/taimoorahmadta)
```

Comment:
AFAIK, your name must be in line 1961, after Taha Latief.
Let us know if you need any help with that. 🙂

Reply:
Thank you,
I have made the changes🙂